sernet.gs.service/src/sernet/gs/scraper/GSScraper.java

```diff
@@ -243,7 +243,14 @@ private void createBrokenRoleReplacements() {
 		} catch (XPathException e) {
 			Logger.getLogger(GSScraper.class).error(e);
 			throw new GSServiceException(e);
-		}
+        } catch (GSServiceException e) {
+            if (LOG.isDebugEnabled()) {
```

Comment:
Please log message and stack trace. Moreover use log level _error_ if _kapitel_ is not equals _"b00"_. Use a constant for string _"b00"_.